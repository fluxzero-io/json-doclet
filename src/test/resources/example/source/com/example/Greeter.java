package com.example;

/**
 * Simple greeter example used in smoke tests.
 */
public class Greeter {

    /**
     * Generates a greeting for the provided {@code name}.
     *
     * @param name person to greet
     * @return greeting message
     */
    public String greet(String name) {
        return "Hello, " + name + "!";
    }
}
