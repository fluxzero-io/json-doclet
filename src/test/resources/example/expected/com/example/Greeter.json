{
  "name" : "Greeter",
  "qualifiedName" : "com.example.Greeter",
  "kind" : "class",
  "documentation" : " Simple greeter example used in smoke tests.\n"
}
